src/XIVLauncher.Core/Program.cs

```diff
@@ -383,6 +385,61 @@ private static ISecretProvider GetSecretProvider(Storage storage)
         }
     }
 
+    private static void GetDistro()
+    {
+        try
+        {
+            if (!File.Exists("/etc/os-release"))
+            {
+                Distro = "ubuntu";
+                DistroLong = "Unknown distribution";
+                IsFlatpak = false;
+                return;
+            }
+            var osRelease = File.ReadAllLines("/etc/os-release");
+            var name = "";
+            var pretty = "";
+            var distro = "";
+            var flatpak = false;
+            foreach (var line in osRelease)
+            {
+                var keyValue = line.Split("=", 2);
+                if (keyValue.Length == 1) continue;
+                if (keyValue[0] == "NAME")
+                    name = keyValue[1];
+                if (keyValue[0] == "PRETTY_NAME")
+                    pretty = keyValue[1];
+                if (keyValue[0] == "ID_LIKE")
+                {
+                    if (keyValue[1].Contains("arch"))
+                        distro = "arch";
+                    if (keyValue[1].Contains("fedora"))
+                        distro = "fedora";
```

Comment:
I don't appear to have `ID_LIKE` on Fedora Silverblue, not sure if it'd get properly set to Fedora because of this?

Reply:
Are you actually using the rpm on silverblue? I thought the idea with that was to use flatpaks. In any case, if you could post the contents of your /etc/os-release file, I'll see about adding it.

Reply:
I use both depending on which one is causing me less problems at the time.

```
NAME="Fedora Linux"
VERSION="38.20230630.0 (Silverblue)"
ID=fedora
VERSION_ID=38
VERSION_CODENAME=""
PLATFORM_ID="platform:f38"
PRETTY_NAME="Fedora Linux 38.20230630.0 (Silverblue)"
DEFAULT_HOSTNAME="fedora"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=38
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=38
VARIANT="Silverblue"
VARIANT_ID=silverblue
. . . removed un-needed values
```